packages/lesswrong/components/localGroups/LocalGroupPage.tsx

```diff
@@ -522,6 +552,8 @@ const LocalGroupPage = ({ classes, documentId: groupId }: {
         {tbdEventsList}
 
         {pastEventsList}
+        
+        {(!isEAForum && (isAdmin || isGroupAdmin)) && <LocalGroupSubscribers groupId={groupId}/>}
```

Comment:
I'd suggest not forum-gating this one, since it's a feature that has been requested on our side :)